pkg/controller/perconaservermongodb/ssl.go

```diff
@@ -209,6 +211,23 @@ func (r *ReconcilePerconaServerMongoDB) createSSLManualy(cr *api.PerconaServerMo
 	return nil
 }
 
+func getShardingSans(clusterName, namespace, clusterSuffix string) []string {
+	return []string{
+		clusterName + "-mongos",
+		clusterName + "-mongos" + "." + namespace,
+		clusterName + "-mongos" + "." + namespace + "." + clusterSuffix,
+		"*." + clusterName + "-mongos",
+		"*." + clusterName + "-mongos" + "." + namespace,
+		"*." + clusterName + "-mongos" + "." + namespace + "." + clusterSuffix,
+		clusterName + "-cfg",
```

Comment:
We have `api.ConfigReplSetName` const.
Maybe should be added same for mongos

Reply:
yea, good point
fixed in 
af56d17